spec/v1/digest_set.md

```diff
@@ -130,20 +133,51 @@ $ printf 'Hello' | git hash-object -t blob --stdin
 
 ### Guidelines
 
-It is RECOMMENDED to use at least `sha256` for compatibility between
-producers and consumers, unless a different hash algorithm is more
-conventional (e.g. `gitCommit` for git).
+It is RECOMMENDED to use at least `sha256` for compatibility between producers
+and consumers, unless a different hash algorithm is more conventional (e.g.
+`gitCommit` for git).
 
-Consumers MUST only accept algorithms that they consider secure and MUST
-ignore unrecognized or unaccepted algorithms. For example, most
-applications SHOULD NOT accept "md5" because it lacks collision resistance.
+Consumers MUST only accept algorithms that they consider secure and MUST ignore
+unrecognized or unaccepted algorithms. For example, most applications SHOULD NOT
+accept "md5" because it lacks collision resistance.
 
-Two DigestSets SHOULD be considered matching if ANY acceptable field
-matches.
+Two DigestSets SHOULD be considered matching if ANY acceptable field matches.
 
 New algorithms MUST document how the value is encoded, e.g. URL-safe base64,
 lowercase hex, etc...
 
+### Use cases for non-cryptographic, immutable, digests
+
+Sometimes users have a need to refer to something by some other immutable
+identifier. Either because the content can't be hashed traditionally, because
+it's impractical to hash traditionally, or because they interact with the
+content through an interface that doesn't expose them to the entirety of the
+content.
+
+In these situations users may wish to use other identifiers in a DigestSet.
+Those users should be careful to understand the trust that they're placing in
+the identifier to be sure that it meets their needs.
```

Comment:
Some editing suggestions to improve readability. Feel free to accept partially or to edit further.

```suggestion
While cryptographic digests are the strongly recommended immutable identifier,
users might have need to refer to an artifact by some other means. For example,
it might be technically infeasible to compute a digest over the content, or
because the user might interact with the content through an interface that
doesn't expose them to the entirety of the content.

In these situations, users MAY use a non-cryptographic identifier in a DigestSet so
long as the risk of the object being mutated is acceptable for the application.
```

Reply:
Done